reviews.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="" />
+    <title>PlaylistPulse</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <main>
+        <h2 class="categories"> Reviews</h1>
```

Comment:
This h2 Reviews title can be in the main html file so when you scroll it stays in place instead of getting scrolled away